lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -411,7 +413,10 @@ instance Arbitrary TxMetadata where
     shrink = shrinkTxMetadata
 
 instance Arbitrary Coin where
-    arbitrary = Coin <$> choose (1, 100000)
+    arbitrary = genStrictlyPositiveCoin
+
+genStrictlyPositiveCoin :: Gen Coin
+genStrictlyPositiveCoin = Coin <$> choose (1, 100_000)
```

Comment:
Again, we should consider replacing this with calls to `genCoinLargePositive` (or similar).

Reply:
Fixed in bce20553d283b5a13c5eeccb76e19d6f88aa7e16.